CMakeLists.txt

```diff
@@ -386,19 +386,15 @@ endif()
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 if(CMAKE_SYSTEM_NAME MATCHES "Darwin")
-  if(ON_APPLE_M1)
```

Comment:
Since this is the only place to use `ON_APPLE_M1`, plz remove it all together.

https://github.com/facebookincubator/velox/blob/6c7bcd7ccc24734c8e2d18029bc47fb1a24f5722/CMakeLists.txt#L295

Reply:
It is used in velox/CMake/resolve_dependency_modules/folly/CMakeLists.txt